app/templates/image-set/cv-results.hbs

```diff
@@ -0,0 +1,44 @@
+<h3>CV Results</h3>
+
+<div class='row'>
+  <div class='col-sm-4'>
+    <ul class='list-group'>
+      {{#each model as |cvResult|}}
+        {{lg-cv-result-summary cvResult=cvResult action='setActiveCvResult'}}
+      {{/each}}
+    </ul>
+  </div>
+
+  <div class='col-sm-4'>
+    <div class='row'>
+      {{#unless searchImageSet.lion}}
+        {{#if showLionNameField}}
+          {{input type='text' name='lionName' placeholder='Lion Name' value=lionName}}
+          <button class='btn btn-xs btn-warning create-lion' {{action 'createLion'}}>
+            Create Lion
```

Comment:
I assume this is beside the point, but the `btn-warning` class is probably not appropriate for this, right? That would be for a cancel/reset type of button, I would expect.